doc/reference/trd104-syscalls.md

```diff
@@ -654,6 +658,8 @@ class. It differs in two ways:
 1. The buffer it passes to the kernel is read-only, and the process MAY
    freely read the buffer.
 2. The kernel MUST NOT write to a buffer shared with a Read-Only Allow.
+3. The *allow number*s in the Read-Only Allow
+are independent from those in the Read-Write Allow.
```

Comment:
The line beginning with `are` should be indented to line up with the word `The` on the previous line (see entry `1.` in this list).